types/audit-details.d.ts

```diff
@@ -116,7 +122,8 @@ declare global {
 
       export interface TableItem {
         debugData?: DebugData;
-        [p: string]: undefined | ItemValue | ItemValue[];
+        subRows?: TableSubRows;
```

Comment:
Can we just call these `subItems` at this point?

Reply:
> Can we just call these subItems at this point?

👍 Makes sense to me, not sure if others were involved in the original bike shedding and will want to weigh in

Reply:
`subRows => subItems` for the detail type, and `subRows => subItemsHeading / subHeading` for the `headings` object?

@patrickhulce @paulirish